### ISSUE TITLE
websocket URL is not considering the URL_PATH.

### ISSUE THREAD

**Issue by cinco** [pentacent/keila#189]
the websocket URL is not considering the URL_PATH.

In the console I get `mydomain.com/live/websocket?_csrf_token=....`, even though I have `URL_PATH=/newsletter`.

While you would expect the websocket at `mydomain.com/newsletter/live/websocket?_csrf_token=....`

URL_PATH was implemented from request feature #77

**Comment by wmnnd** [comment]
Thanks for reporting this. The problem comes from the fact that the socket path is hardcoded in `app.js`. I’ll fix it for the next release.

**Comment by cinco** [comment]
thanks @wmnnd 

The projects link in the left menu is also ignoring URL_PATH.
![image](https://user-images.githubusercontent.com/125763/216778990-a960a4dd-1168-428c-a1fe-7712d6d3ced9.png)

Should I open another issue or can it be handled in this one?

**Comment by wmnnd** [comment]
I’ll also handle it in this issue.

**Comment by wmnnd** [comment]
This issue has been fixed in af6e37e00fad3e46991540ad5e88e8058b7da6d3. The fix will be released as part of the upcoming next version with the Block editor.

**Comment by st3xupery** [comment]
Forgive me, but was this issue fixed by the commit? I am running the docker image for keila 0.14.4 and I am still getting this exact same error when accessing the Campaign editor.

**Comment by wmnnd** [comment]
@st3xupery Did you set `URL_HOST` as well?

**Comment by st3xupery** [comment]
Yes, and most all of the assets are querying from the correct URL_HOST and URL_PATH, it only shows up for this websocket route (and a route for some fonts).

![image](https://github.com/pentacent/keila/assets/5591230/0bbe3828-ed5e-428d-969f-fc312696ede1)

**Comment by Marthaarman** [comment]
> Yes, and most all of the assets are querying from the correct URL_HOST and URL_PATH, it only shows up for this websocket route (and a route for some fonts).
> 
> ![image](https://private-user-images.githubusercontent.com/5591230/329259355-0bbe3828-ed5e-428d-969f-fc312696ede1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMyOTM3NTAsIm5iZiI6MTcyMzI5MzQ1MCwicGF0aCI6Ii81NTkxMjMwLzMyOTI1OTM1NS0wYmJlMzgyOC1lZDVlLTQyOGQtOTY5Zi1mYzMxMjY5NmVkZTEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgxMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MTBUMTIzNzMwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MTMzNGY1Yjk2MzQwNzI2ODA0MDJiMmY3NzRmNWNiMmU4MzViZTU5ZTc5YTA0ZTI5YWI1ZDIwNDI2OWI1YTFhMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.22rN6qzbH2dSL9GkWU0oxdZDFp69ztJv_00epzrVG6c)

Have you found a solution to this?
